apkid/rules/apk/packers.yar

```diff
@@ -27,6 +27,25 @@
 
 include "common.yar"
 
+rule dexprotector : packer
+{
+ /**
+ * DexProtector v6.x.x Both Standard And Business Edition And Demo // Edited By Jasi2169
```

Comment:
Could you add https://dexprotector.com/ to this comment? It's nice to know.
Also, you could put "Jasi2169" as the metadata.author if you prefer. No other rules use that yet but if we ever make a feature that uses it, it's a better place to store that info than in a comment.